programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,4 @@
+function solution(seoul) {
+  var answer = seoul.findIndex((v) => (v === "Kim"))
```

Comment:
크으... ```indexOf()```를 쓰지않고 ```findIndex()```를 사용한 점 칭찬합니다.
```suggestion
  const answer = seoul.findIndex((v) => (v === "Kim"))
```